pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -80,6 +80,10 @@ let
         substitute ${neovim}/share/applications/nvim.desktop $out/share/applications/nvim.desktop \
           --replace 'TryExec=nvim' "TryExec=$out/bin/nvim" \
           --replace 'Name=Neovim' 'Name=WrappedNeovim'
+
+				# add icon to display
+				mkdir -p $out/share/pixmaps
```

Comment:
regroup this mkdir with the previous one maybe ?

Reply:
@teto ,
Changes added. Please review.